docs/generated/reference/manifest.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
This manifest.json file shouldn't be committed (at least, we don't commit it in either the docker based or bazel based docgen pipeline)